src/markdown/handbook/tools/jira/issues/webinar.md

```diff
@@ -11,8 +11,15 @@ Request related to the creation of a specific webinar by the Communication Team.
 
 ### When to create a Webinar request?
 
-- TO DO
+Consider creating a Webinar ticket whenever there's a need to edit a pre-recorded webinar. Generally, webinar requests entail clipping footage, adjusting audio levels, and adding intros and outros. Webinar requests are not for requesting a recording. To request recording assistance, please reach out in our Slack channel ([#d-comm-des](https://liferay.slack.com/archives/CKETJ1DTR)) or [create a Communication Design Request](https://liferay.design/blueprints/jira/design-request/).
 
+### How to create a Webinar request?
+
+Find best practices for creating a Webinar request on:
+* [Confluence](https://liferay.atlassian.net/l/cp/EsZ7BP1X)
+* [Blueprints](https://liferay.design/blueprints/jira/webinar/)
```

Comment:
for future reference — if the URL is in the same domain (in this case — liferay.design), it's better to use a relative path (e.g. `../../jira/webinar`) — that way in the event the base URL changes, the link will still work

Reply:
(BTW the example I used is wrong, don't use that exact URL path 😬 )